vm-setup/roles/common/defaults/main.yml

```diff
@@ -11,7 +11,8 @@ libvirt_nic_model: virtio
 default_disk: 50
 default_memory: 8192
 default_vcpu: 2
-num_nodes: 2
+num_nodes: 1
+num_redfishvmnodes: 1
```

Comment:
confused here, didnt we agree that we will leave the default setup as it is and only create redfishvmnode for live-iso test case? Wouldn't this change mean you are always using 1 normal node and 1 refishvm node regardless of the test ?

Reply:
I changed in common.sh and missed this one, now its fixed.